pkg/cache/v3/status.go

```diff
@@ -93,6 +128,40 @@ type ResponseWatch struct {
 
 	// Subscription stores the current client subscription state.
 	subscription Subscription
+
+	enableStableVersion bool
+
+	fullStateResponses bool
```

Comment:
all fields in this struct are documented, you probably want to preserve that and document those fields.